functions.php

```diff
@@ -39,11 +51,14 @@ function get_sum ($cost) {
     
     return $cost.' ₽';
   };
-
-  
   
+  /**
+  * Возвращает массив со всеми данным таблицы category из БД
+  * 
+  * @param $connect mysqli Ресурс соединения
```

Comment:
```suggestion
  * @param mysqli $connect Ресурс соединения
```
Сначала пишем тип, потом название переменной